tony-core/src/main/java/com/linkedin/tony/Constants.java

```diff
@@ -94,6 +94,9 @@
   public static final String AM_STDOUT_FILENAME = "amstdout.log";
   public static final String AM_STDERR_FILENAME = "amstderr.log";
 
+  public static final String TASK_EXECUTOR_PYTHON_STDERR_FILENAME = "python.err";
+  public static final String TASK_EXECUTOR_PYTHON_STDOUT_FILENAME = "python.out";
```

Comment:
not necessarily Python, we can practically run anything with tony.. need a better name

Reply:
subprocess.err? @oliverhu